Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportToolBar.cpp

```diff
@@ -87,6 +87,13 @@ namespace EMStudio
                     // Send the reset camera event.
                     AnimViewportRequestBus::Broadcast(&AnimViewportRequestBus::Events::ResetCamera);
                 });
+
+            cameraMenu->addSeparator();
+            m_followCharacterAction = cameraMenu->addAction("Follow Character");
+            m_followCharacterAction->setCheckable(true);
+            m_followCharacterAction->setChecked(false);
+            connect(m_followCharacterAction, &QAction::triggered, this, &AnimViewportToolBar::OnFollowCharacter);
```

Comment:
Shall we just use a lambda here? The OnFollowCharacter function is a one-liner. Might be better for code overview.